certified-connectors/Office365GroupsMail/apiProperties.json

```diff
@@ -0,0 +1,50 @@
+{
+  "properties": {
+    "capabilities": [
+      "actions",
+      "triggers"
+    ],
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "<<Client ID placeholder>>",
+          "scopes": [
+            "Group.ReadWrite.All offline_access"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "AzureActiveDirectoryResourceId": "https://graph.microsoft.com"
+          },
+          "customParameters": {
+            "loginUri": {
+              "value": "https://login.windows.net"
+            },
+            "tenantId": {
+              "value": "common"
+            },
+            "resourceUri": {
+              "value": "https://graph.microsoft.com/"
+            }
+          }
+        }
+      },
+      "token:TenantId": {
```

Comment:
Is token:TenantId required to be here?

Reply:
Looks like it is not necessary; I'll remove it.